llvm/test/CodeGen/X86/yk-patchpoint.mir

```diff
@@ -1,58 +1,58 @@
 # RUN: llc -mtriple=x86_64-- --yk-stackmap-add-locs %s -o - | FileCheck %s
 
-# This test verifies that the `--yk-stackmap-add-locs` flag correctly
-# adds additional location information to patchpoint instructions.
+# Check that patchpoint instructions correctly track additional locations for
```

Comment:
With the risk of sounding nitpicky: can we add that we are checking this to make we don't treat patchpoints arguments as tracked locations (which is what we were doing before). And also possibly rename the file to something like `yk-patchpoint-donot-track-args.mir`?

Reply:
oki :point_right: d4ec4859d7236d6a5ebdfaabbd135c009c350f9e